externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/operation/SparkSQLOperationManager.scala

```diff
@@ -50,8 +50,6 @@ class SparkSQLOperationManager private (name: String) extends OperationManager(n
     sessionToSpark.remove(sessionHandle)
   }
 
-  def getOpenSparkSessionCount: Int = sessionToSpark.size()
```

Comment:
Why remove this method, seems useful to test.

Reply:
sgtm